src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -80,18 +104,19 @@ const AnalysisCardHeaders = ({ sort, onSort }) => {
   ])
 }
 
+//TODO: move to separate file
 const AnalysisCard = ({
   currentRuntime, namespace, name, lastModified, metadata, application, workspaceName, onRename, onCopy, onDelete, onExport, canWrite,
   currentUserHash, potentialLockers
 }) => {
   const { lockExpiresAt, lastLockedBy } = metadata || {}
-  const lockExpirationDate = new Date(parseInt(lockExpiresAt))
-  const locked = currentUserHash && lastLockedBy && lastLockedBy !== currentUserHash && lockExpirationDate > Date.now()
+  const isLockExpired = new Date(parseInt(lockExpiresAt)).getTime() > Date.now()
+  const isLocked: boolean = currentUserHash && lastLockedBy && lastLockedBy !== currentUserHash && !isLockExpired
```

Comment:
Again, I'd encourage not mixing in stylistic changes to make large PRs like this easier to review.

Reply:
There was actually a bug here, the compared values are not the same type, so I needed to change this code a bit anyways. I'd agree, but I dislike boolean variables names not proceeded by `is` `should` or `can` immensely :)